### DISCUSSION TITLE
Add manager to PR

### DISCUSSION THREAD

**Discussion by kroese** [renovatebot/renovate#26276] [Suggest an Idea]
### Tell us more.

Often times when I receive PR from Renovate and want to disable them, I run into the problem that I dont know which specific manager I must disable for that type of PR.

If Renovate would put the name of the manager inside the PR, for example ```npm``` or ```cargo```, it was so much easier to know which one to disable.

**Comment by rarkins** [comment]
You can achieve this using https://docs.renovatebot.com/configuration-options/#prbodycolumns and/or https://docs.renovatebot.com/configuration-options/#prbodydefinitions. e.g. you could add a new column just for manager, or you could modify the existing Package value to include manager.

Note that PRs may have multiple updates and therefore multiple managers so manager is a property of update, not PR
